website/snippets/cloud-feature-parity.md

```diff
@@ -0,0 +1,14 @@
+The following table outlines which dbt Cloud features are supported on the different SaaS options available today. For more information about feature availability, please [contact us](https://www.getdbt.com/contact/).
```

Comment:
I wonder how we can keep this table current while in a low maintenance way? I wonder if we should add to the checklist something about updating this table when updating Enterprise features? 🤔